phoenix-core-client/src/main/java/org/apache/phoenix/compile/FromCompiler.java

```diff
@@ -280,7 +280,7 @@ public static ColumnResolver getResolver(NamedTableNode tableNode, PhoenixConnec
 
     public static ColumnResolver getResolver(SingleTableStatement statement, PhoenixConnection connection)
             throws SQLException {
-        SingleTableColumnResolver visitor = new SingleTableColumnResolver(connection, statement.getTable(), true);
```

Comment:
Do we use one of these resolvers when we run `ALTER TABLE <table-name> SET PROPERTY` statement?

Reply:
Yes. 
Confirmed from SetPropertyIT that setting property on table/view goes through `ExecutableAddColumnStatement` and that uses the resolver change we made.